system-test/compute.js

```diff
@@ -934,7 +934,7 @@ describe('Compute', () => {
     });
   });
 
-  describe('vms', () => {
+  describe.only('vms', () => {
```

Comment:
we should make sure we remove this `.only`.

Reply:
Fixed.